pysoundcomparisons/__main__.py

```diff
@@ -658,7 +655,8 @@ def write_valid_soundfolders(args):
         if len(data) > 0:
             for row in data:
                 valid_folders.add(row['FilePathPart'])
-    with codecs.open("valid_soundfolders.txt", "w", "utf-8-sig") as f:
+    with codecs.open(api.repos.joinpath(
+            "soundfiles", "valid_soundfilepaths.txt"), "w", "utf-8-sig") as f:
         f.write("\n".join(sorted(valid_folders)))
         f.close()
```

Comment:
When using the context manager (i.e. within the `with` block), there's no need to call `close()` explicitly.